### PR TITLE
#1660: Fix ExcelControl copying of stringified attribute

### LINKED ISSUE #1659: ExcelControl(stringified=True) documentation could clarify interaction with schemas
# Overview

[`formats.excelControl.stringified`](https://framework.frictionlessdata.io/docs/formats/excel.html#reference-formats.excelcontrol.stringified) is documented as "Stringifies all the cell values.".

However it appears that it doesn't actually do that, even in the test cases that were added as part of the PR that added the `stringified` option.

e.g.  in [`def test_xls_parser_cast_int_to_string_1251_xlsx()`](https://github.com/frictionlessdata/frictionless-py/pull/1259/files#diff-8fb31016a58c40a13d350a23e361ffef7761b55a4d1bb8d2a6823d774c1f3a64R131) the test is:

```python
resource = Resource(descriptor, control=formats.ExcelControl(stringified=True))
assert resource.read_rows() == [
    {"A": "001", "B": "b", "C": "1", "D": "a", "E": 1},
    {"A": "002", "B": "c", "C": "1", "D": "1", "E": 1},
]
```

where the `"E": 1` is a number not a string.

Or an even simpler reproduction:

```python
from frictionless import Resource, formats

res = Resource(path="data/table-infer-boolean.xlsx", control=formats.ExcelControl(stringified=True))
print(res.read_rows())
```
prints:
```
[{'number': Decimal('1.0'), 'string': 'two', 'boolean': True}, {'number': Decimal('3.0'), 'string': 'four', 'boolean': False}]
```

Is this the intended behaviour and I am misunderstanding the purpose of `stringified=True` or is this a bug (which I am happy to look at fixing).  

To be honest, the code looks like it should convert everything to strings, but it doesn't seem to according to the unit tests (and our tests). Perhaps @roll or @shashigharti might remember from their work on [the PR that created this capability](https://github.com/frictionlessdata/frictionless-py/pull/1259)?

### LINKED ISSUE #1660: `ExcelControl` attribute `stringified` gets lost during `to_copy()`
# Overview

The `stringified` attribute of the `ExcelControl` gets reset to the default value (False) when the control is copied using `to_copy()` (or the resource it is attached to is copied causing it to be copied).  This appears to be because it was missed from the `metadata_profile_patch` definition.

## To reproduce
```python
from frictionless import formats

original = formats.ExcelControl(stringified=True)

copy = original.to_copy()

print("original:", original.stringified, ", copy:", copy.stringified)

# Returns "original: True , copy: False"
```

I'll submit a PR to fix this shortly.

### PR DESCRIPTION
**Pull request by richardt-engineb**
# Description

As described in #1660, the `stringified` attribute is not copied during the `to_copy()` as it is not included in the `metadata_profile_patch`.

This commit fixes it by adding it to the `metadata_profile_patch` and updates the documentation to clarify the interaction of `stringified` and schemas.  This also inclues a unit test to verify the bug fix.

- fixes #1660 
- fixes #1659 

# Tests

Ensure the new unit test detects the issue before the fix:

```
> pytest frictionless/formats/excel/__spec__/test_control.py::test_excel_control_to_copy

[...]

>       assert control_copy == control_with_changed_attributes
E       AssertionError: assert {'type': 'exc...tError': True} == {'type': 'exc...tError': True}
E
E         Omitting 8 identical items, use -vv to show
E         Differing attributes:
E         ['stringified']
E
E         Drill down into differing attribute stringified:
E           stringified: False != True

frictionless/formats/excel/__spec__/test_control.py:25: AssertionError
```

Ensure the unit test runs and passes after the fix:

```
> pytest frictionless/formats/excel/__spec__/test_control.py::test_excel_control_to_copy -v

[...]

frictionless/formats/excel/__spec__/test_control.py::test_excel_control_to_copy PASSED [100%]
```

### CODE REVIEW

**Review by pierrecamilleri — APPROVED** [review]
Thanks for the pull request.

**Comment by pierrecamilleri** [comment]
I am not sure why the CI checks were not performed. @richardt-engineb, would you mind pushing an empty commit to see if it triggers the CI ?

**Comment by richardt-engineb** [comment]
Trivial comment update commit added. Hopefully this will trigger it!

Also might need approval from a maintainer as I'm a first time contributor from a public fork.

**Comment by pierrecamilleri** [comment]
It worked :) Thanks again

**Pull request merged**
